rocAL/rocAL/source/meta_data/meta_node_crop.cpp

```diff
@@ -40,8 +40,8 @@ void CropMetaNode::update_parameters(pMetaDataBatch input_meta_data, pMetaDataBa
     _crop_height = _meta_crop_param->croph_arr;
     _x1 = _meta_crop_param->x1_arr;
     _y1 = _meta_crop_param->y1_arr;
-    _input_width_val = _meta_crop_param->in_width;
-    _input_height_val = _meta_crop_param->in_height;
+    // _input_width_val = _meta_crop_param->in_width;   TODO - Commenting for now to be replaced with ROI
```

Comment:
This change is going to be updated in next PR's?

Reply:
Yes